File: js/programmers/level.0/prgm_120890_1.js
function solution(array, n) {
  let array2 = [];
  let array3 = [];
  for (i in array) {
    array2.push(Math.abs(array[i] - n));
  }
  for (j = 0; j < array2.length; j++) {
    if (array2[j] === Math.min(...array2)) {
      array3.push(array[j]);
    }
  }
  return Math.min(...array3);
}

File: python/programmers/level.0/prgm_120890_1.py
def solution(array, n):
    arr = []
    arr2 = []

    for i in array:
        arr.append(abs(n-i))
    
    for j in range(len(arr)):
        if arr[j] == min(arr):
            arr2.append(array[j])

    return min(arr2)